portal-web/test/functional/com/liferay/portalweb/macros/Permissions.macro

```diff
@@ -63,6 +63,15 @@ definition {
 			userLoginFullName = "${userLoginFullName}");
 	}
 
+	macro addPermissionViaAPI {
+		JSONPermission.addPermission(
+			actionId = "${resourceAction}",
+			groupName = "${groupName}",
+			name = "${resourceName}",
+			roleTitle = "${roleTitle}",
+			roleType = "${roleType}");
+	}
+
```

Comment:
To assign permission to a role via API there is already `Permissions.definePermissionViaJSONAPI()`.
Please use that one and remove this macro. Thanks!